packages/circuit-ui/components/CurrencyInput/CurrencyInput.tsx

```diff
@@ -67,6 +67,8 @@ const DEFAULT_FORMAT = {
   groupDelimiter: ',',
 };
 
+const DUMMY_DELIMITER = '?';
```

Comment:
Why `?`?

Reply:
:eyes:

Reply:
No particular reason, it was a random pick.

Reply:
:pray: